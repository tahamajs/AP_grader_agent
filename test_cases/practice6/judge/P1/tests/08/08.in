POST signup ? username "low_mist" password "meow"
POST logout ?
POST signup ? username "cool_man" password "1234"
POST logout ?
POST signup ? username "amir_akbar" password "1382"
GET reserves ?
POST reserve ? restaurant_name "sib" table_id "2" start_time "16" end_time "19" foods "pitza,burger"
GET reserves ? reserve_id "1"
POST logout ?
POST login ? username "cool_man" password "1234"
POST logout ?
POST login ? username "low_mist" password "meow"
POST reserve ? restaurant_name "sib" table_id "2" start_time "19" end_time "21" foods "pitza,pitza,burger"
GET reserves ? restaurant_name "sib" reserve_id "1"
POST logout ?
POST login ? username "cool_man" password "1234"
POST reserve ? restaurant_name "hani" table_id "1" start_time "18" end_time "21" foods "joojeh,joojeh,cholo morgh"
GET reserves ? restaurant_name "hani" reserve_id "7"
GET reserves ? restaurant_name "hani" reserve_id "1"
POST logout ?
POST login ? username "amir_akbar" password "1382"
POST reserve ? restaurant_name "burger land" table_id "1" start_time "14" end_time "15"
GET reserves ?
GET reserves ? restaurant_name "sib"
POST logout ?