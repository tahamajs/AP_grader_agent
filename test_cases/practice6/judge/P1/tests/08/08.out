OK
OK
OK
OK
OK
Empty
Reserve ID: 1
Table 2 for 16 to 19 in sib
Price: 560
Bad Request
OK
OK
OK
OK
Reserve ID: 2
Table 2 for 19 to 21 in sib
Price: 910
Permission Denied
OK
OK
Reserve ID: 1
Table 1 for 18 to 21 in hani
Price: 760
Not Found
1: hani 1 18-21 cholo morgh(1) joojeh(2)
OK
OK
Reserve ID: 1
Table 1 for 14 to 15 in burger land
Price: 0
1: burger land 1 14-15
1: sib 2 16-19 burger(1) pitza(1)
1: sib 2 16-19 burger(1) pitza(1)
OK